arkane/ess/gaussianTest.py

```diff
@@ -198,11 +198,11 @@ def test_load_number_scans(self):
         log = GaussianLog(os.path.join(self.data_path, 'isobutanolQOOH_scan.log'))
         self.assertAlmostEqual(log._load_number_scans(), 36)
 
-    def test_determine_qm_software(self):
+    def test_ess_factory(self):
```

Comment:
Good catch, thanks! I made this a minor commit

Reply:
Squash that minor commit with this one, since it fixes code introduced in this PR (i.e. don't let this code enter the git history just to remove it in a later commit).